src/solver/solve.jl

```diff
@@ -242,21 +242,19 @@ function check_conservation(sol)
         ) / energy_total
 
     mass_conservation =
-        (sum(sol.u[end].c.ρ) - sum(sol.u[1].c.ρ)) / sum(sol.u[1].c.ρ)
+        (sum(sol.u[end].c.ρ) - sum(sol.u[1].c.ρ)) / sum(sol.u[begin].c.ρ)
```

Comment:
```suggestion
        (sum(sol.u[end].c.ρ) - sum(sol.u[begin].c.ρ)) / sum(sol.u[begin].c.ρ)
```
Also @dennisYatunin prefers 1 over begin. I don't have a preference as long as we are consistent:)